workingspacebooking/src/main/java/naitei/group5/workingspacebooking/repository/VenueRepository.java

```diff
@@ -20,7 +23,7 @@ public interface VenueRepository extends JpaRepository<Venue, Integer> {
             """)
     Optional<Venue> findByIdWithDetails(@Param("id") Integer id);
 
-    List<Venue> findByOwnerId(Integer ownerId);
-
-    List<Venue> findByVerified(Boolean verified);
+    // Tìm tất cả venue của 1 owner, load kèm venueStyle
+    @EntityGraph(attributePaths = {"venueStyle"})
+    List<Venue> findByOwner_Id(Integer ownerId);
```

Comment:
```suggestion
    List<Venue> findByOwnerId(Integer ownerId);
```
trong java dùng camelCase nha